### ISSUE TITLE
Different directory orders

### ISSUE THREAD

**Issue by alper** [planetary-social/planetary-ios#89]
I could imagine it being useful at this stage to have directory ordering by date joined and number of followers etc.

**Comment by rabble** [comment]
Yep, it's also on our list to combine the directory we host with all the accounts you've found through your follows and other pubs.

**Comment by rabble** [comment]
Look at using CoreML to sort the contacts in the directory.

**Comment by rabble** [comment]
Look at using CoreML to sort the contacts in the directory.

**Comment by rabble** [comment]
Look at using CoreML to sort the contacts in the directory.

**Comment by rabble** [comment]
Look at using CoreML to sort the contacts in the directory.

**Comment by rabble** [comment]
Look at using CoreML to sort the contacts in the directory.

**Comment by danlatorre** [comment]
Well need to do tests to see how different results look like, and test w/ users. Also what's the outcome here, is this about increasing ways to find existing/new friends?
